frontend/js/src/user/Dashboard.tsx

```diff
@@ -106,31 +106,34 @@ export default function Listen() {
 
   const queryClient = useQueryClient();
 
-  const receiveNewListen = (newListen: string): void => {
-    let json;
-    try {
-      json = JSON.parse(newListen);
-    } catch (error) {
-      toast.error(
-        <ToastMsg
-          title="Coudn't parse the new listen as JSON: "
-          message={error?.toString()}
-        />,
-        { toastId: "parse-listen-error" }
-      );
-      return;
-    }
-    const listen = formatWSMessageToListen(json);
-
-    if (listen) {
-      setWebSocketListens((prevWebSocketListens) => {
-        return [
-          listen,
-          ..._.take(prevWebSocketListens, maxWebsocketListens - 1),
-        ];
-      });
-    }
-  };
+  const receiveNewListen = React.useCallback(
+    (newListen: string): void => {
+      let json;
+      try {
+        json = JSON.parse(newListen);
+      } catch (error) {
+        toast.error(
+          <ToastMsg
+            title="Coudn't parse the new listen as JSON: "
```

Comment:
```suggestion
            title="Couldn't parse the new listen as JSON: "
```
Was already there before, but still